internal/xds/translator/route.go

```diff
@@ -1,17 +1,27 @@
 package translator
 
 import (
+	core "github.com/envoyproxy/go-control-plane/envoy/config/core/v3"
 	route "github.com/envoyproxy/go-control-plane/envoy/config/route/v3"
 	matcher "github.com/envoyproxy/go-control-plane/envoy/type/matcher/v3"
 
 	"github.com/envoyproxy/gateway/internal/ir"
 )
 
 func buildXdsRoute(httpRoute *ir.HTTPRoute) (*route.Route, error) {
```

Comment:
can you also add some tests here https://github.com/envoyproxy/gateway/blob/791408311703b662f56f26fcadf2b2e77154f34a/internal/xds/translator/translator_test.go#L26 for redirect and direct response ?